Makefile

```diff
@@ -5,6 +5,14 @@
 # capture the current date we build the application from
 BUILD_DATE = $(shell date +%Y-%m-%dT%H:%M:%SZ)
 
+# Versions installed for OpenSSH and SSHPass binaries.
+# This is the ONLY place these hardcoded versions are set.
+# They're used in the Dockerfile the GitHub Actions workflow,
+# the integration tests, and the static build flags for Go.
+# Note: No space between the equals and the value else issues arise.
+OPENSSH_VERSION=8.8_p1-r1
+SSHPASS_VERSION=1.09-r0
```

Comment:
I wonder if we can/should combine this with https://docs.renovatebot.com/modules/datasource/#repology-datasource to configure renovate to be aware of the versioning to give us a kick in the butt when there are patches/fixes and such? in the same token, i have never used that setup so not sure how well it works.

Reply:
Ohhh that's cool, didn't know about that feature. I'll take a look at it tomorrow to see about integrating this. My goal tomorrow/this-week is to get a tagged release and to start I just wanted pinned versions, but something longer term would be like this. So when I merge this PR I'll open a couple of issues to track things like this and the movement away from depending on vela-kaniko for anything.